app/.eslintrc.js

```diff
@@ -64,6 +64,7 @@ module.exports = {
         "@typescript-eslint/no-unused-vars": "error",
         // The usage of `any` defeats the purpose of typescript. Consider using `unknown` type instead instead.
         "@typescript-eslint/no-explicit-any": "error",
+        "@typescript-eslint/await-thenable": "off",
```

Comment:
ℹ️ Wasn't sure the best option here. 

We get compile errors in next 15 that we need to await params. 

Doing so resolves the compile errors but causes this lint error to fire. Quick google search didn't turn up any results, so went with this for now.